mne/viz/backends/_pyvista.py

```diff
@@ -230,7 +230,15 @@ def scene(self):
         return self.figure
 
     def set_interaction(self, interaction):
-        getattr(self.plotter, f'enable_{interaction}_style')()
+        if interaction == "rubber_band":
+            for renderer in self.plotter.renderers:
+                renderer.enable_parallel_projection()
+            style = vtk.vtkInteractorStyleRubberBand2D()
```

Comment:
Can you open a PR to pyvista to add this as a mode? Then we can `try/except` or `hasattr` here

Reply:
(For example, this mode will suffer from the "click-and-drag-not-captured" bug, because the fix for this is in PyVista subclassing the VTK interactor style and dealing with button presses.)

Reply:
> Can you open a PR to pyvista to add this as a mode? 
Sure, will do.

I use middle-click to drag with this one. I was not aware of this bug.

Reply:
https://github.com/pyvista/pyvista/issues/116

Reply:
(It's possible we also need to capture middle clicks, in which case hopefully from https://github.com/pyvista/pyvista/pull/854 it's clear what you'd need to do to capture those, too)